torch/distributed/_tensor/op_schema.py

```diff
@@ -71,14 +71,18 @@ def __str__(self) -> str:
         if self.input_specs is None:
             input_specs_str = ""
         else:
-            input_specs_str = ", ".join(
-                [
-                    self.pretty_print_placements(spec.placements)
-                    for spec in self.input_specs
-                ]
+            input_specs_str = (
+                "("
+                + ", ".join(
+                    [
+                        self.pretty_print_placements(spec.placements)
+                        for spec in self.input_specs
+                    ]
+                )
+                + ") -> "
             )
         output_spec_str = self.pretty_print_placements(self.output_spec.placements)
-        return f"({input_specs_str}) -> ({output_spec_str}) @ mesh layout: {tuple(self.output_spec.mesh.mesh.shape)}"
+        return f"{{{input_specs_str}{output_spec_str}}}"
```

Comment:
curious: any reason we want to have the curly braces? i.e. why do we prefer `{(input1, input2) -> output}` over`(input1, input2) -> output`?

Reply:
I don't quite remember why lol, maybe the later is also good, I'll try to change this in follow up PRs.